pipenv/cli.py

```diff
@@ -209,7 +209,7 @@ def do_install_dependencies(dev=False, only=False, bare=False, requirements=Fals
     else:
         if not bare:
             click.echo(crayons.yellow('Installing dependencies from Pipfile.lock...'))
-        with open(project.lockfile_location, 'r') as f:
+        with open(project.lockfile_location) as f:
```

Comment:
It looks like this, and possibly other areas are being read in as bytes instead of strings now. This is breaking the build for Python 3.3 and 3.4.